src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefConditionalOperatorTests.cs

```diff
@@ -918,6 +918,142 @@ .maxstack  1
         }
 
         [Fact]
+        public void TestRefConditionalHomelessBranches()
+        {
+            var source = @"
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var o = new C1();
+            var o1 = new C1();
+
+            (args != null ? ref o.field : ref o1.field).ToString();
+
+            System.Console.WriteLine(o.field.value);
+        }
+    }
+
+    class C1
+    {
+        public readonly S1 field;
+    }
+
+    struct S1
+    {
+        public int value;
+
+        public override string ToString()
```

Comment:
Nit: test would be clearer if you changed this to `void Mutate()`

Reply:
Resolved: this also tests override.

---
In reply to: [139494331](https://github.com/dotnet/roslyn/pull/21982#discussion_r139494331) [](ancestors = 139494331)